SpectraLogic.EscapePodClient.Integration.Test/EscapePodClientErrorTests.cs

```diff
@@ -291,6 +291,132 @@ public void HeadArchiveErrorTests()
             Assert.IsFalse(EscapePodClientFixture.EscapePodClient.IsArchiveExist("not_found"));
         }
 
+        //TODO can be tested after ESCP-160 is resolved
```

Comment:
I believe 160 is already resolved.  Can this code be un-commented out?

Reply:
it is in the new PR